run.sh

```diff
@@ -25,7 +26,10 @@ print-help() {
     echo "     ASIC to be tested"
     echo "  -t TARGET"
     echo "     Target device with this NPU"
+    echo "  -c [start|stop]"
```

Comment:
The initial idea behind these helper scripts was to have the same commands as we have when we operate with the Dockers - `build`, `run`, `exec`. So, using `run` to stop the Docker does not fit very well. Also, do we want to just stop or to remove?

Reply:
You can remove docker if you use -r option during start.
Do we really want to have one more script stop.sh just for stopping dockers?
Maybe we can merge start/stop functions (even maybe include tb_ctl.sh) to the same script, e.g. sai-c-ctl.sh?

Reply:
In such case `-i` option does not make sense. Also, in case we decide to build a topology - run multiple server nodes and 1..n client nodes - start/stop atomic operations will be more useful.

The approach with the separate small scripts is definitely not the best and at some point we may even end-up with a kind of library (bash or python or..) with the basic operations. But I'd prefer to postpone this decision until we introduce a topology creation infra.

Reply:
Agree. So I'll keep it as is for now. Later we will decide what is the right approach